aiogram/dispatcher/handler.py

```diff
@@ -30,6 +34,7 @@ def __init__(self, dispatcher, once=True, middleware_key=None):
         self.once = once
 
         self.handlers = []
+        self.specs = {}
```

Comment:
Why do you place specs to dictionary instead of extending items in `self.handlers`?